core/src/main/java/de/bwaldvogel/mongo/wire/MongoDatabaseHandler.java

```diff
@@ -129,7 +129,7 @@ private MongoReply queryFailure(MessageHeader header, MongoServerException excep
         return new MongoReply(header, obj, ReplyFlag.QUERY_FAILURE);
     }
 
-    private Document handleCommand(Channel channel, MongoQuery query) {
+    Document handleCommand(Channel channel, MongoQuery query) {
```

Comment:
this seem ok to you? i have zero experience with testing Netty applications - it didn't appear there was a real clear way to wire up `Channel`s and `ChannelHandlerContext`s

Reply:
I was hoping we could formulate it as case in `AbstractBackendTest`… I just tried by myself and failed. So I will merge the PR for now. If we find a better solution to test it, we can improve it anytime.